examples/minimal_svg_renderer/src/main.ts

```diff
@@ -1,6 +1,5 @@
-import { parseDot } from "@msagl/parser"
-import { RendererSvg } from "@msagl/renderer"
-
+import {parseDot} from '@msagl/parser'
+import {RendererSvg} from '@msagl/svgRenderer'
```

Comment:
Thinking about convenience to avoid less typing. If we use names renderer-svg, renderer-webGL, renderer-common then the intellisense provides a longer list compared to svg-renderer, webGL-renderer, and common-renderer.  What is the argument for using the renderer-* pattern?

Reply:
renamed

Reply:
fixed